### PR TITLE
Gdb 6765 implement add remove nodes from cluster

### PR DESCRIPTION
**Pull request by yordanalexandrov**
* implemented action buttons, that are only visible if node is leader, for adding and removing nodes actions
* implemented modal dialogs with own controllers for both functions
* refactored to be able to select nodes from remote locations
* changed visualization to match create cluster dialog

### CODE REVIEW

File: src/js/angular/clustermanagement/controllers.js [hunk 1]
```diff
@@ -548,3 +642,73 @@ function AddLocationCtrl($scope, $modalInstance, toastr, productInfo, $translate
...
+AddNodesDialogCtrl.$inject = ['$scope', '$modalInstance', 'data', '$modal', 'RemoteLocationsService'];
+
+function AddNodesDialogCtrl($scope, $modalInstance, data, $modal, RemoteLocationsService) {
+    const clusterConfiguration = angular.copy(data.clusterConfiguration);
+    const clusterModel = angular.copy(data.clusterModel);
+    $scope.nodes = [];
+
+    $scope.clusterNodes = clusterModel.locations.filter((location) => clusterConfiguration.nodes.includes(location.rpcAddress));
+    $scope.locations = clusterModel.locations.filter((location) => !$scope.clusterNodes.includes(location));
+    $scope.locations.forEach((location) => location.isNew = true);
```
Comment (svilenvelikov) [inline src/js/angular/clustermanagement/controllers.js:655]: Can't you chain this with the filter above?

File: src/js/angular/clustermanagement/templates/modal/add-nodes-dialog.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<!-- Start add nodes to cluster modal -->
```
Comment (svilenvelikov) [inline src/js/angular/clustermanagement/templates/modal/add-nodes-dialog.html:1]: what are these comments?!

File: src/js/angular/clustermanagement/templates/modal/remove-nodes-dialog.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" ng-click="cancel()">{{'common.cancel.btn' | translate}}</button>
+        <button id="wb-remove-nodes-from-cluster-submit" class="btn btn-primary"
+                ng-disabled="!nodesToRemoveCount || (clusterNodes.length - nodesToRemoveCount < 2)"
```
Comment (svilenvelikov) [inline src/js/angular/clustermanagement/templates/modal/remove-nodes-dialog.html:38]: Can you expose a variable in the controller for this `clusterNodes.length - nodesToRemoveCount < 2` and bind to it, then reuse it few lines above.

File: src/js/angular/clustermanagement/controllers.js (hunk 1 above)
Comment (yordanalexandrov) [reply src/js/angular/clustermanagement/controllers.js:655]: No, as the consumer does not return the array.

File: src/js/angular/clustermanagement/templates/modal/add-nodes-dialog.html (hunk 2 above)
Comment (yordanalexandrov) [reply src/js/angular/clustermanagement/templates/modal/add-nodes-dialog.html:1]: I saw them somewhere else. They are for consistency :)

Comment (yordanalexandrov) [reply src/js/angular/clustermanagement/templates/modal/add-nodes-dialog.html]: removed them

**Pull request merged**